docs/BUILD.md

```diff
@@ -46,4 +46,4 @@ After the release build status is **success** ensure that the new version is pub
 ### Checklist 
 
 - Publish completed without an issue.
-- The new version is available in the (https://repo1.maven.org/maven2/com/commercetools/commercetools-sync-java)
\ No newline at end of file
+- The new version is available in the (https://repo1.maven.org/maven2/com/commercetools/commercetools-sync-java)
```

Comment:
What's the difference here?

Reply:
No diff. Not sure why it is shown here on github.